<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parking</class>
 <widget class="QWidget" name="Parking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Парковка</string>
  </property>
  <property name="accessibleName">
   <string>Парковка</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Место</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="place">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Количество мест</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="space_count">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Тариф</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tarif">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Количесвто занятых мест</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="occuped_count">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Количество свободных мест</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="free_spaces_count">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Количество уехавших машин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="exit_count">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_set">
      <property name="text">
       <string>Машина приехала</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_remove">
      <property name="text">
       <string>Машина уехала</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
